mne_bids/tests/test_read.py

```diff
@@ -133,6 +135,42 @@ def test_handle_events_reading():
     events, event_id = mne.events_from_annotations(raw)
 
 
+def test_handle_coords_reading():
+    """Test reading coordinates from a
+    BIDS electrodes.tsv and coordsystem.json files."""
```

Comment:
docstring for tests should be preferably one line ... at least the first line should not wrap over. It's a PEP8 rule I think